libs/api/domains/air-discount-scheme/src/lib/api-domains-air-discount-scheme.service.ts

```diff
@@ -65,7 +65,11 @@ export class AirDiscountSchemeService {
       if (isValid) {
         discounts.push({
           ...discount,
-          user: { ...relation, name: relation.firstName },
+          user: {
+            ...relation,
+            name: relation.firstName,
+            fund: discount.user.fund,
```

Comment:
Not in the case of explicit discounts.
The explicit discount fund bypasses requirements validation by design.
Therefore we need to overwrite the user generated fund in this case.

It will do nothing in the case of normal discounts but will allow explicit codes to function as expected.

This does raise a valid point though and I'll put in a task to rewrite, if possible, the logic such that an overwrite is not needed or the necessity at least reduced.